config/newrelic.yml

```diff
@@ -0,0 +1,67 @@
+# This file configures the New Relic Agent.  New Relic monitors Ruby, Java,
+# .NET, PHP, Python, Node, and Go applications with deep visibility and low
+# overhead.  For more information, visit www.newrelic.com.
+#
+# Generated October 28, 2022
+#
+# This configuration file is custom generated for NewRelic Administration
+#
+# For full documentation of agent configuration options, please refer to
+# https://docs.newrelic.com/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration
+
+common: &default_settings
+  # Required license key associated with your New Relic account.
+  # Use NEW_RELIC_APP_NAME env variable to specify the license key
+
+  # Your application name. Renaming here affects where data displays in New
+  # Relic.  For more details, see https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/renaming-applications
+  # Use NEW_RELIC_APP_NAME env variable to customize the application name
+  app_name: "Open Food Network"
+
+  distributed_tracing:
+    enabled: true
+
+  # To disable the agent regardless of other settings, uncomment the following:
+
+  # agent_enabled: false
```

Comment:
I think we probably want to disable the agent by default if there's no API key present (in development for example). I think this file allows interpolation with erb.

Maybe here we can do something like:
```ruby
agent_enabled: <%= ENV["NEW_RELIC_LICENSE_KEY"].present? %>
```

Reply:
Good idea.

Reply:
the new relic config doesn't allow interpolation any more. I couldn't find a way to disable the agent if the license key is not present, but I updated the config to disable the agent in dev, test and staging.

Reply:
That's a problem. Not every production instance will have a new relic key. Do we set it to false everywhere and then use environment variables to control it instead?

Reply:
Yes I think that would be a better solution. Do we need to update any documentation around this ?

Reply:
> Do we need to update any documentation around this ?

Yes.

* The ofn-install repository has a template for the secrets which needs entries with comments.
* The wiki describes optional services, how to set them up etc. It needs to be mentioned there.